index.js

```diff
@@ -31,8 +31,8 @@ const main = async () => {
   // it impossible to properly match to jobs returned from the API.
   // Since we later match to only failed jobs, it's not a problem for now.
   const filteredJobs = response.data.jobs.filter(
-      (job) => (job.status == 'completed' && !ignoreJobsRegex ||
-        !job.name.match(ignoreJobsRegex)),
+      (job) => (job.status == 'completed' && (!ignoreJobsRegex ||
+        !job.name.match(ignoreJobsRegex))),
```

Comment:
this should ignore `in_progress` jobs like itself with status `null` 👍

Reply:
that's exactly the point 😂